src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -136,6 +137,41 @@
   WebSocketBase writeTextMessage(String text);
 
   /**
+   * Writes a ping to the connection. This will be written in a single frame, so the contents of data should be small.
+   * This method should not be used to write application data and should only be used for implementing a keep alive or
+   * to ensure the client is still responsive, see RFC 6455 Section 5.5.2.
+   *
+   * Ping frames must be less than 126 bytes (octets).
+   *
+   * There is no pingHandler because RFC 6455 section 5.5.2 clearly states that the only response to a ping is a pong
+   * with identical contents.
+   *
```

Comment:
add line break tag (p)

Reply:
I don't see line breaks after the final paragraph in other docs (for example `binaryHandlerID`.  Should there be trailing line breaks?